setup.py

```diff
@@ -21,41 +17,39 @@
 
 packages = ["genomepy", "genomepy/plugins"]
 
-package_data = {"genomepy": ["cfg/*.yaml"]}
+# this replaces the PyPa MANIFEST.in
+package_data = {"genomepy": ["cfg/*.yaml"], "": ["LICENSE", "README.md"]}
+
+entry_points = {"console_scripts": ["genomepy=genomepy.cli:cli"]}
 
 requires = [
-    "pytest",
     "click",
     "pyfaidx>=0.5.1",
     "norns>=0.1.5",
     "xmltodict",
     "bucketcache",
     "requests",
+    "biopython>=1.73",
     "appdirs",
+    "psutil",
 ]
 
-entry_points = {"console_scripts": ["genomepy=genomepy.cli:cli"]}
-
 classifiers = [
     "Development Status :: 4 - Beta",
     "Intended Audience :: Developers",
     "Intended Audience :: Science/Research",
     "License :: OSI Approved :: MIT License",
     "Operating System :: MacOS :: MacOS X",
-    "Operating System :: MacOS :: MacOS X",
     "Operating System :: POSIX :: Linux",
     "Programming Language :: Python",
-    "Programming Language :: Python :: 3.3",
-    "Programming Language :: Python :: 3.4",
-    "Programming Language :: Python :: 3.5",
-    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3",
     "Topic :: Scientific/Engineering :: Bio-Informatics",
 ]
 
 setup(
     name="genomepy",
     version=__version__,  # noqa: F821
-    description="Genomes in Python",
+    description="Download genomes in Python",
```

Comment:
Maybe we need to change this. You can download genomes, but that's not specific to Python, and you can also *use* these genomes easily in Python. Come up with not too long sentence that covers this?

Reply:
how about "Downloading & processing of genomes in command line and Python"?